_data/glossary/annotation.yaml

```diff
@@ -0,0 +1,9 @@
+id: annotation
+name: Annotation
+full-link: docs/concepts/overview/working-with-objects/annotations
+tags:
+- fundamental
+short-description: >
+  An annotation is a key-value pair and used to attach arbitrary non-identifying metadata to objects.
```

Comment:
"An annotation is" can be dropped to avoid repeating the term.

Reply:
Not only can it be dropped, it should be dropped. This will also let you tighten even further:
`A key-value pair that is used to attach arbitrary non-identifying metadata to objects."

Reply:
Thanks @tonglil and @Bradamant3  sounds great.